certs/variables.tf

```diff
@@ -74,8 +75,14 @@ variable "s3_bucket_prefix" {
   default     = ""
 }
 
-variable "cidr_whitelist" {
+variable "vpn_whitelist" {
```

Comment:
Just additional granularity if desired, as the SSH should really be an admin backdoor only.